### PR TITLE
Substitute ID references in href attributes 

### PR DESCRIPTION
**Pull request by Kreeg**
Initial PR #577 

Closes #577

### CODE REVIEW

**Comment by XhmikosR** [comment]
This is wrong @Kreeg. Needs a proper rebase

**Comment by Kreeg** [comment]
@XhmikosR yeah I kinda freaked out too when I saw that :D

File: lib/svg-sprite/shape.js [hunk 1]
```diff
@@ -712,10 +712,10 @@ SVGShape.prototype.setNamespace = function(ns) {
             });
 
             // Substitute ID references in href attributes
-            select('//@href', this.dom).forEach(href => {
+            select('//@href[not(@xlink:href)]', this.dom).forEach(href => {
```
Comment (XhmikosR) [inline lib/svg-sprite/shape.js:715]: @Kreeg are these changes correct? They are included in a commit that mentions only the tests.

Comment (Kreeg) [reply lib/svg-sprite/shape.js]: @XhmikosR they're correct, but obsolete, I changed these back to previous version.

**Pull request merged**